src/backend/distributed/utils/maintenanced.c

```diff
@@ -215,6 +215,7 @@ CitusMaintenanceDaemonMain(Datum main_arg)
 	Oid databaseOid = DatumGetObjectId(main_arg);
 	MaintenanceDaemonDBData *myDbData = NULL;
 	TimestampTz nextStatsCollectionTime USED_WITH_LIBCURL_ONLY = GetCurrentTimestamp();
```

Comment:
What if we initialized this to `GetCurrentTimestamp` + one minute? Do you think that would reduce false positive failures due to extension/loading problems?

Reply:
Done.